src/Symfony/Component/Notifier/Bridge/Expo/ExpoOptions.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Expo;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Imad ZAIRIG <https://github.com/zairigimad>
+ *
+ * @see https://docs.expo.dev/push-notifications/sending-notifications/
+ *
+ * @experimental in 5.4
+ */
+final class ExpoOptions implements MessageOptionsInterface
+{
+    private $to;
+
+    /**
+     * @see https://docs.expo.dev/push-notifications/sending-notifications/#message-request-format
+     */
+    protected $options;
+
+    private $data;
+
+    public function __construct(string $to, array $options = [], array $data = [])
+    {
+        $this->to = $to;
+        $this->options = $options;
+        $this->data = $data;
+    }
+
+    public function toArray(): array
+    {
+        return array_merge(
+            $this->options,
+            [
+                'to' => $this->to,
+                'data' => $this->data,
+            ]
+        );
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return $this->to;
+    }
+
+    public function title(string $title): self
```

Comment:
please add `@return $this` annotations every where possible

Reply:
done